### PR TITLE
Edit pass on Blazor pages, routing, layouts eBook chapter

### PR DESCRIPTION
**Pull request by scottaddie**

### CODE REVIEW

File: docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-Blazor also supports deep linking, where the browser makes a request to a specific route other than the root of the app. Requests for deep links sent to the server are routed to the Blazor app, which then routes the request client-side to the correct component.
+Blazor also supports *deep linking*. Deep linking occurs when the browser makes a request to a specific route other than the root of the app. Requests for deep links sent to the server are routed to the Blazor app, which then routes the request client-side to the correct component.
 
-A simple page in ASP.NET Web Forms might look like this:
+A simple page in ASP.NET Web Forms might contain the following markup:
 
 *Name.aspx*
 
-```html
+```aspx
```
Comment (Youssef1313) [inline docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md]: I think it should be aspx-csharp ?

File: docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md [hunk 2]
```diff
@@ -212,17 +218,17 @@ In Blazor, you handle page layout using layout components. Layout components inh
 
 When the page with a layout is rendered, the page is rendered within the contents of the specified layout at the location where the layout renders its `Body` property.
 
-To apply a layout to a page, use the `@layout` directive.
+To apply a layout to a page, use the `@layout` directive:
 
 ```razor
```
Comment (Youssef1313) [inline docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md]: I haven't seen `razor` identifier before in docs. I doubt that such an identifier is supported.
/cc @mairaw

Comment (mairaw) [reply docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md]: When it's not supported by default, docs will just display that identifier with no coloring.

File: docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md (hunk 1 above)
Comment (mairaw) [reply docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md]: yep, it should

File: docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md (hunk 2 above)
Comment (scottaddie) [reply docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md]: It's not currently supported. I submitted a request to engineering a few months back. The closest dev lang to it is "CSHTML".

**Pull request merged**